Testing/WinMLRunnerTest/WinMLRunnerTest.cpp

```diff
@@ -747,45 +801,45 @@ namespace WinMLRunnerTest
             const std::wstring command = BuildCommand({ EXE_PATH, L"-model", modelPath, L"-input", inputPath });
             Assert::AreEqual(HRESULT_FROM_WIN32(ERROR_INVALID_PARAMETER), RunProc((wchar_t *)command.c_str()));
         }
-        TEST_METHOD(ProvidedCSVInputGPUSaveTensor)
-        {
+        TEST_METHOD_WITH_NAME(ProvidedCSVInputSaveTensor)
             const std::wstring modelPath = CURRENT_PATH + L"SqueezeNet.onnx";
             const std::wstring inputPath = CURRENT_PATH + L"fish.csv";
+            const std::wstring tensorDataPath = TENSOR_DATA_PATH + L"\\" + METHOD_NAME;
             const std::wstring command = BuildCommand({ EXE_PATH, L"-model", modelPath, L"-input", inputPath,
-                                                        L"-SaveTensorData", L"First", TENSOR_DATA_PATH, L"-GPU" });
-            Assert::AreEqual(S_OK, RunProc((wchar_t*)command.c_str()));
-            Assert::AreEqual(true, CompareTensors(L"OutputTensorData\\Squeezenet_fish_input_GPU.csv",
-                                                  TENSOR_DATA_PATH + L"\\softmaxout_1GpuIteration1.csv"));
+                                                        L"-SaveTensorData", L"First", tensorDataPath, L"-GPU" });
+            Assert::AreEqual(S_OK, RunProc((wchar_t *)command.c_str()));
+            Assert::AreEqual(true, CompareTensors(L"softmaxout1GpuIteration1.csv",
```

Comment:
Output file should still remain OutputTensorData\\Squeezenet_fish_input_GPU.csv

Reply:
Definitely some kind of merge or intermediate issue.  Fixed.